client/state/selectors/get-current-plan-purchase-id.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get, isNaN } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+
+/**
+ * Returns a site's current plan purchase ID or null if the site doesn't exist or the purchase ID
+ * is unknown.
+ *
+ * The ID returned by this selector is retrieved from the site plan. It is intended to provide a
+ * means of retrieving full purhcase information based on a site and it's plan information.
+ * Caution! It _does not_ retrieve the ID from a purchase.
```

Comment:
This comment is confusing to me - isn't the selector retrieving the ID of the current plan purchase, which essentially is retrieving the ID of a purchase?